menu/menu_displaylist.c

```diff
@@ -924,9 +950,14 @@ static int menu_displaylist_parse_playlist(menu_displaylist_info_t *info,
           *   no further action is necessary */
 
          if (string_is_empty(entry->label))
+         {
             fill_short_pathname_representation(menu_entry_label, entry->path, sizeof(menu_entry_label));
+         }
          else
+         {
             strlcpy(menu_entry_label, entry->label, sizeof(menu_entry_label));
+            (*sanitization)(menu_entry_label, sizeof(menu_entry_label));
```

Comment:
This is pedantic, but the compiler may not (probably won't) optimise away a function pointer that references an empty function.

Basically, when label display mode is `LABEL_DISPLAY_MODE_DEFAULT`, we want the absolute minimum performance impact possible - so there should be no function call at all.

I would suggest the following:

- In the `switch (playlist_get_label_display_mode(playlist))` statement, the `default` should set `sanitization` to `NULL`

- In this `else` condition it should be:

```c
if (sanitization)
   (*sanitization)(menu_entry_label, sizeof(menu_entry_label));
```